tools/separate_asm.py

```diff
@@ -25,6 +25,10 @@
   # seek a pattern like (e.ENVIRONMENT), which is in the shell.js if-cascade for the ENVIRONMENT override
   import re
   m = re.search('\((\w+)\.ENVIRONMENT\)', everything)
+  if not m:
+    m = re.search('(\w+)\.arguments = \[\];', everything)
+  if not m:
+    m = re.search('(\w+)\.arguments=\[\];', everything)
```

Comment:
Am I reading this right and it's saying match `xxx.arguments = [];`, with or without whitespace around the `=`?
How about merging these into something like
`m = re.search('(\w+)\.arguments\s*=\s*\[\];', everything)`

Reply:
Good idea, done.